src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -53,19 +58,30 @@ const createTestBulkUpload = async (
 	return bulkUpload;
 };
 
-const createTestBaseFields = async (): Promise<void> => {
-	await db.sql<BulkUpload>('baseFields.insertOne', {
+const createTestBaseFields = async (): Promise<[BaseField, BaseField]> => {
```

Comment:
Fun Fact: it seems this is the way you say "it is returning an array of size 2" in typescript!